src/index.html

```diff
@@ -1,14 +1,50 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="logo">
+        <a href="/home" class="logo__img"></a>
+      </div>
+      <nav class="navbar">
+        <ul class="nav__list">
+          <li class="nav__link">
+            <a href="/apple" class="nav__link-link is-active">Apple</a>
+          </li>
+          <li class="nav__link">
+            <a href="/samsung" class="nav__link-link">Samsung</a>
+          </li>
+          <li class="nav__link">
+            <a href="/smartphones" class="nav__link-link">Smartphones</a>
+          </li>
+          <li class="nav__link">
+            <a href="/laptops+computers" class="nav__link-link"
+            data-qa="hover">Laptops & Computers</a>
```

Comment:
Checklist p13
![image](https://github.com/mate-academy/layout_moyo-header/assets/84296041/6572ec2d-cb41-4619-bc48-4ad6e68b257b)

```suggestion
            <a 
              href="/laptops+computers" 
              class="nav__link-link"
              data-qa="hover"
            >
              Laptops & Computers
            </a>
```